### ISSUE TITLE
README wording suggestion

### ISSUE THREAD

**Issue by sf-mrichter** [github/site-policy#27]
In the README, under the few things that should not be posted in the repo, it says this:

Please don't give other users legal advice, to avoid confusion.

I suggest rewording that to be:

To avoid confusion, please don't give other users legal advice.

It could easily be read to mean that giving others legal advice to avoid confusion is not a good idea.  Technically, this is true, but it's not what you mean and the above wording is unambiguous.

**Comment by bean5** [comment]
I still think it is ambiguous, actually. To be fully unambiguous, perhaps:

Please don't give other users legal advice. Legal advice tends to confuse users.

OR

Please don't give advice because it tends to confuse them.

OR

To avoid confusing users, please don't give legal advice.


What do you think? In any case, mind making a small PR for your proposal for a speedier acceptance and application (if approved)?

**Comment by sf-mrichter** [comment]
I don’t see the sit-policy branch.  Where should I create the PR?

**Comment by sf-mrichter** [comment]
Scratch that (I gotta read the pages more carefully…).

Which branch of site-policy should I use?


From: Mark Richter
Sent: Monday, July 17, 2017 10:23
To: 'github/site-policy' <reply@reply.github.com>; github/site-policy <site-policy@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [github/site-policy] README wording suggestion (#27)

I don’t see the sit-policy branch.  Where should I create the PR?

**Comment by bean5** [comment]
I do but know either.
